pages/content/amp-dev/events/amp-roadshow.html

```diff
@@ -63,13 +63,78 @@ <h2 class="ap-o-stage-content-subline">Come join us in your city for a free, inc
 
   <div class="ap-o-cities ap--content-wide">
     <h3 class="ap-o-cities-headline">Cities</h3>
-    <div class="ap-m-jumbo ap-m-jumbo-cities">
-      <div class="ap-a-ico">
+
+    {% if doc.pod.env.dev %}
+    {% do doc.styles.addCssFile('css/components/molecules/tip.css') %}
+    {% do doc.icons.useIcon('icons/map.svg') %}
+    <div class="ap-m-tip ap-m-tip-note">
+      <div class="ap-m-tip-content">
+        <div class="ap-a-ico ap-m-tip-icon">
+          <svg><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#map"></use></svg>
+        </div>
+        <h4>Want to add cities?
+        <button
+          on="tap:AMP.setState({coordsGridToggled: !coordsGridToggled})"
+          [text]="coordsGridToggled ? 'Hide grid' : 'Show grid'"
+          type="button">
+          Show grid
+        </button>
+        </h4>
+        <div hidden [hidden]="!coordsGridToggled">
+          <strong><code>Coordinates: <span [text]="coords"></span></code></strong>
+          <p>You can copy this coordinates and use it in the <strong>roadshow.yaml</strong> <em>(pages/shared/data/roadshow.yaml)</em></p>
```

Comment:
Little nit: "You can copy this coordinates and use it in the" -> "You can copy these coordinates and use them in"

Reply:
Thanks for this. Fixed it.